test/ctr.bats

```diff
@@ -807,6 +807,46 @@ function teardown() {
 	stop_crio
 }
 
+@test "ctr with default list of capabilities from crio.conf" {
+	start_crio
+	run crictl runp "$TESTDATA"/sandbox_config.json
+	echo "$output"
+	[ "$status" -eq 0 ]
+	pod_id="$output"
+	run crictl create "$pod_id" "$TESTDATA"/container_redis.json "$TESTDATA"/sandbox_config.json
+	echo "$output"
+	[ "$status" -eq 0 ]
+	ctr_id="$output"
+	run crictl exec --sync $ctr_id grep Cap /proc/1/status
+	echo "$output"
+	[ "$status" -eq 0 ]
+	[[ "$output" =~ 00000000a80425fb ]]
+
+	cleanup_ctrs
+	cleanup_pods
+	stop_crio
+}
+
+@test "ctr with list of capabilities given by user in crio.conf" {
+	start_crio "" "" "" "--default-capabilities CHOWN --default-capabilities KILL" ""
```

Comment:
I think we should do `--default-capabilities=CHOWN,KILL` instead of one argument per capability.